using-valtimo/plugin/object-token-authentication/configure-plugin.md

```diff
@@ -0,0 +1,27 @@
+# Using the Object Token Auhentication Plugin
+
+The Object Token Auhentication plugin is used to provide authentication for ZGW (Zaakgericht Werken) plugins that connect to APIs which 
+use a token as authentication. Currently the plugins that this plugin can provide authentication for are the following:
+
+- [Objecten API](../objecten-api/configure-plugin.md)
+- [Objecttypen API](../objecttypen-api/configure-plugin.md)
+
+### Prerequisites
+
+This page requires you to have knowledge on how to configure Objecten API
```

Comment:
This page requires knowledge on how to configure Objecten API. For more information, see [here](whateverthelocationis).

Reply:
This might be difficult. We don't have a page with more information.